.github/workflows/build-docker-image-and-binaries.yaml

```diff
@@ -64,9 +71,14 @@ jobs:
           ARCH: ${{ matrix.arch }}
           OS_TYPE: ${{ matrix.os }}
           SEMVER: ${{ github.event.inputs.tag }}
+          WASM: ${{ github.event.inputs.WASM }}
+          CGO_ENABLED: 0
         run: |
+          if [ "${{ github.event.inputs.cgo }}" == "true" ]; then
+            export CGO_ENABLED=1
+          fi
           OS=$(echo "$OS_TYPE" | sed -e 's/ubuntu-20.04/linux/; s/macos-latest/darwin/')
-          make build-binaries-multiarch
+          make WASM="${WASM}" CGO_ENABLED="${CGO_ENABLED}" build-binaries-multiarch
```

Comment:
Do you really need to do this? Aren't they already set as env vars and therefore the process should have access to them?

Reply:
I thought the same thing, but I decided to stay consistent with what we already do here:

https://github.com/axelarnetwork/axelar-core/pull/2039/files#diff-dd2c0eb6ea5cfc6c4bd4eac30934e2d5746747af48fef6da689e85b752f39557R34
https://github.com/axelarnetwork/axelar-core/pull/2039/files#diff-e78e82e598899ded6250f8bfd8970463c877cc0e15e2f6080f985d641635015fL18
https://github.com/axelarnetwork/axelar-core/pull/2039/files#diff-6b171362cce648d7aa9cc8781f4bec2322332a58402574641632852b0ef889d0L18

Reply:
i confirmed that declaring them as arguments in `make` is not needed indeed. Updated the PR accordingly